src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -282,7 +282,7 @@ MUST be present:
 | **Column name** | **Requirement level** | **Description**                     |
 | --------------- | --------------------- | ----------------------------------- |
 | name            | REQUIRED              | Name of the electrode.              |
-| x               | REQUIRED              | Recorded position along the x-axis. |
+| x               | REQUIRED              | Recorded position along the x-axis.   |
```

Comment:
this should be covered by the [remark-preset-lint-markdown-style-guide](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-preset-lint-markdown-style-guide) rule called [`remark-lint-table-pipe-alignment`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-table-pipe-alignment) ... but somehow this issue is not detected like it was before ...